src/tables/HVAR.js

```diff
@@ -0,0 +1,78 @@
+import r from 'restructure';
+import {resolveLength} from 'restructure/src/utils';
+
+
+let F2DOT14 = new r.Fixed(16, 'BE', 14);
```

Comment:
nit more than anything else, but as a JS developer I'm always curious: why `let` rather than `var` (since we're in top level scope and `let` is specifically for block scoping) or `const` (since these structs are not getting reassigned later they're more constants than variables)? 

(I've seen people explain this in other codebases as "jslint complains because it doesn't like mixed var keywords", which is a bit of a strange, but also completely understandable reason, just curious about it in this codebase)